### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by x-tabdeveloping**
I made an executive decision to move all paper-related things into a separate folder, and remove them from the library.

#### Why did I do this?
1. I would like to use this repo and package for future projects.
2. I think the library code got quite messy and overly coupled. This happened, because I initially introduced some abstractions, that did not work well with the matplotlib and seaborn plots that you guys produced, and workarounds and exceptions had to be used to get them to work.
3. We had 4 extra plotting dependencies, and I didn't feel comfortable having them in the package.

#### What did I do?
1. I removed all plotting code from the library.
2. I made the table command more generic, so that it can be used in future projects.
3. I moved everything $S^3$ related to a separate folder and included detailed instructions for how to reproduce and what.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,15 +63,34 @@ python3 -m topic_benchmark run -e intfloat/e5-large-v2
 
 The results for each embedding model will be found in the `results` folder (unless a value for `--out_file` is explicitly passed).
 
-Each of these commands will compute results for all metrics, all datasets, all topic modeling methods, and all hyperparameters (i.e., number of topics) reported in the paper given a specific embedding model. Note that running these commends can be therefore be very time consuming, due to the large number of models that are estimated and evaluated.
+To produce figures and tables in the paper, you can use the scripts in the  `s3_paper_scripts/` folder.
+```bash
+pip install -r s3_paper_scripts/requirements.txt
+
+# Table 3: Main Table (tables/main_table.tex)
+python3 s3_paper_scripts/effect_of_preprocessing.py
```
Comment (rbroc) [inline README.md]: this should be `python3 s3_paper_scripts/main_table.py`, right?

**Review by rbroc — COMMENTED** [review]
I've added one comment to the code. One additional minor suggestion could be to start the reproducibility section of the README by saying something along the lines of "results from the paper are available in the `results` folder, but to reproduce them from scratch do the following: ..." (this is all to pre-chew information for inattentive reviewers). The rest all looks and sounds sane to me, I am assuming you have ensured it all runs. ;)

File: README.md (hunk 1 above)
Comment (x-tabdeveloping) [reply README.md]: Yes

**Comment by x-tabdeveloping** [comment]
I haven't confirmed that these work yet, but I'm on it rn.

**Pull request merged**